robottelo/cli/hostcollection.py

```diff
@@ -35,25 +37,42 @@ class HostCollection(Base):
     command_base = 'host-collection'
 
     @classmethod
+    def transform_ids(cls, options=None):
+        """Workaround host unification feature not being completed to use host
+        id instead content host uuid
```

Comment:
The only thing i'm not sure about is your approach - basically if list of uuids is send then it will transform it into ids, if ids passed - then it will pass it as it is. But when list contains both uuids and ids (i can barely image that, but maybe we'll have some big end to end scenario, where list of hosts will be gathered through the test and at some point user will accidentaly collect both uuids and ids in the same list... yeah, not a big chance of that), in such case this func will ignore all the ids and pass only transformed uuids, which is bad and hard to debug why...
We could go smarter and work with any kind of list, for example like that:
```python
host_ids = []
for host_id in options.get('host-ids').split(','):
    host_ids.append(
        host_id if host_id.is_numeric() else 
        Host.info({
            'name': ContentHost.info({'id': host_id})['name'].lower()
        })['id']
    )
if host_ids:
    options['host-ids'] = ','.join(host_ids)
```
Not sure if that worth it though as the chance of failure is pretty low, but on the other hand why should we leave hidden bug here?